TypeCobol.Test/Parser/Programs/Cobol85/MissingSpaceAfterDotDecimalPointIsComma.rdzMix.txt

```diff
@@ -0,0 +1,71 @@
+﻿Line 1[8,22] <37, Warning, General> - Warning: "END PROGRAM" is missing.
+       IDENTIFICATION DIVISION.
+      *Warning : A blank was missing before character "P" in column 19.
+      *          A blank was assumed.
+Line 4[18,18] <12, Warning, Tokens> - A blank was missing before character "P" in column 19. A blank was assumed.
+       PROGRAM-ID.Pgm.
+      * OK: No warning
+       AUTHOR.AUTHOR.
+      * OK: No warning
+       DATE-WRITTEN.NOVEMBRE 80.
+      * OK: No warning
+       DATE-COMPILED..
+       ENVIRONMENT DIVISION.
+       CONFIGURATION SECTION.
+      *Warning: A blank was missing before character "I" in column 24.
+      *         A blank was assumed.
+Line 15[23,23] <12, Warning, Tokens> - A blank was missing before character "I" in column 24. A blank was assumed.
+       SOURCE-COMPUTER.IBM-370.
+      *Warning: A blank was missing before character "I" in column 24.
+      *         A blank was assumed.
+Line 18[23,23] <12, Warning, Tokens> - A blank was missing before character "I" in column 24. A blank was assumed.
+       OBJECT-COMPUTER.IBMCOBOLII.
+      *Warning: A blank was missing before character "D" in column 22.
+      *         A blank was assumed.
+Line 21[21,21] <12, Warning, Tokens> - A blank was missing before character "D" in column 22. A blank was assumed.
+       SPECIAL-NAMES.DECIMAL-POINT IS COMMA.
+      *Error: "."was invalid. Scanning was resumed at the next area
+      *       "A" item, level-number or the start of the next clause.
+      *Warning: A blank was missing before character "." in column 22.
+      *         A blank was assumed.
+Line 26[21,21] <12, Warning, Tokens> - A blank was missing before character "." in column 22. A blank was assumed.
+Line 26[22,22] <27, Error, Syntax> - Syntax error : extraneous input '.' expecting {ProgramIdentification, ProgramEnd, ClassIdentification, MethodEnd, ProcedureDivisionHeader, FileSectionHeader, WorkingStorageSectionHeader, LocalStorageSectionHeader, LinkageSectionHeader, FunctionDeclarationEnd, GlobalStorageSectionHeader}
+       data division..
+      *Error: "."was invalid. Scanning was resumed at the next area
+      *       "A" item, level-number or the start of the next clause.
+      *Warning: A blank was missing before character "." in column 32.
+      *         A blank was assumed.
+Line 31[31,31] <12, Warning, Tokens> - A blank was missing before character "." in column 32. A blank was assumed.
```

Comment:
So we didn't get any error here as comment suggests on line 34?

Reply:
Yes this is another issue. I forgot to remove this line in the test. @delevoye Can you remove lines 34 and 35 
```
      *Error: "."was invalid. Scanning was resumed at the next area
      *       "A" item, level-number or the start of the next clause.
```

Reply:
Lines have been removed from here, the cbl file and the other similar test.